### PR TITLE
Added tooltip on star icon

### PR DESCRIPTION
**Pull request by hardiksharma11**
I have added the tooltip on the start icon

**Behaviour**

 -When starIcon is selected it says `Remove from wishlist` 
 -When it is not selected it says `Add to wishlist`

### CODE REVIEW

**Comment by zekth** [comment]
sorry we recently changed the build system, we use `pnpm` now, so please run `pnpm i` and update the lock file for it

File: src/pages/index.js [hunk 1]
```diff
@@ -52,6 +53,7 @@ const IndexPage = () => {
   }, getFavoriteMakers());
 
   const sortedMakers = sortBy(data.allSitePage.nodes, (n) => !favoriteMakers.includes(n.pageContext.maker.id));
+  console.log(sortedMakers)
```
Comment (theosenoussaoui) [inline src/pages/index.js:56]: You can also reomve this before merging.

**Review by theosenoussaoui — COMMENTED** [review]
As @zekth said, you can remove the yarn lockfile before merging.

**Comment by hardiksharma11** [comment]
Done the required changes ...

**Pull request closed without merging**